packages/lib/gpcircuits/circuits.json

```diff
@@ -137,6 +137,7 @@
       "ownerV4EntryIndex",
       "ownerV4IsNullifierHashRevealed",
       "numericValueEntryIndices",
+      "numericValueInRange",
```

Comment:
Why was this missing before only in this entry and not the other ones?

Reply:
Your guess is as good as mine!